client/src/TenderClient.js

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+
+class TenderClient {
```

Comment:
If we have a class, we will need to instantiate it every time we want to make a request. It would be easier to use if we just export the functions